### PR TITLE
Add viral transcript detection heatmap

### PR DESCRIPTION
**Pull request by sharlagelfand**
Finally ready to add in our first big feature @scottkall ! 

This is the viral transcript detection: heatmap from RNAseq data feature. You can see the feature here: https://proxe.shinyapps.io/PRoXe_alpha2/ under Liquid Tumors -> Liquid Tumor PDX Viral Transcript Detection, and it sounds like Mark has done a bunch of thorough exploration and testing.

Let me know if you have any questions or if you spot anything off! Thanks.

### CODE REVIEW

**Comment by scottkall** [comment]
Hi @sharlagelfand! Great job! Sorry it took me 12 days to get to this. In the future, creating pull requests is great, but if you're on a deadline feel free to merge if things seem to be working, especially if I'm not responsive.

First impressions of things that could be improved:

1. The heatmap could be sorted so that names are on the left side of the screen and so that yellow values are on the left. These are really the only samples of interest. This will also solve the problem of labels moving out of view when you expand the plot width.
2. I'm not the biggest fan of this color scheme -- and I know you're drawing from my own. Just something to keep in mind going forward.
3. There's no default in 'Select WHO Classification'.
4. I really wish I could also filter by some row and column value information -- filter out low samples, for instance. Or Mark has mentioned binarizing the data, which requires some function / threshold.

Will now poke through the code.

File: compile_upload.R [hunk 1]
```diff
@@ -1,50 +1,48 @@
 ## this script is for pre-compiling data and running/uploading app
 
 # set order of read-in scripts
-read_in_scripts <- c(
-  "clean_liquid.R", "seq.R", "rna_seq.R", "oncoprint2.R",
-  "clean_solid.R", "jax.R", "finalize.R"
-)
+read_in_scripts = c("clean_liquid.R", "clean_virusseq.R", "seq.R","rna_seq.R","oncoprint2.R",
```
Comment (scottkall) [inline compile_upload.R]: @sharlagelfand : I'm noticing that your nicely-spaced reformatting seems to be being overwritten all through this document. How do we want to fix/avoid this?

Comment (sharlagelfand) [reply compile_upload.R]: Thanks, I just reapplied the styling. I think it was because I was working in various branches, but I'll make sure files are styled going forward.

**Comment by sharlagelfand** [comment]
Thanks @scottkall -- if you don't mind I'm going to move these stylistic (non-code) comments over to Wrike so that all of the implementation discussion is in one place (and Mark can chime in too). I'll respond + tag you there.

**Review by scottkall — APPROVED** [review]
@sharlagelfand : changes look good to me. Merging approved. I like how you've factored the app into parts, btw.

**Comment by sharlagelfand** [comment]
Thanks @scottkall! Probably it should actually be in [shiny modules](https://shiny.rstudio.com/articles/modules.html) but in lieu of learning those yet, this organization is at least easier to go through :)

**Comment by scottkall** [comment]
Also, let me know if you need me to actually do the merge, @sharlagelfand. I haven't done it yet because  a) you're free to (if you can) and b) I thought there was a chance you might still be adding things.

**Comment by sharlagelfand** [comment]
No worries, I can do it myself! Thanks. I added the styling but I think that's good now. I'll do the merge and deploy later this week
